### PR TITLE
feat : added products to product-page.

### PR DESCRIPTION
**Pull request by shuklahimanshu81**
Added products to product-page.

See deploy preview here : https://deploy-preview-3--blossom-store.netlify.app/products-page/products-page.html

### CODE REVIEW

File: products-page/products-page.css [hunk 1]
```diff
@@ -41,6 +42,36 @@
...
+    opacity: 0.5;
+}
+
+.text-overlay{
+     bottom:50%;
+} 
+
+.card-badge{
+    position:static;
+    margin-top: -100%;
```
Comment (dev-enforced) [inline products-page/products-page.css]: This line of code looks little confusing? Can you explain why we are using it

File: products-page/products-page.css [hunk 2]
```diff
@@ -41,6 +42,36 @@
...
+}
+
+.products-number{
+    font-size: calc(var(--text-size-m) - 8px);
+}
+
+.products{
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-gap: 25px;
```
Comment (dev-enforced) [inline products-page/products-page.css:62]: Can use rem too depending on your requirements

**Review by dev-enforced — APPROVED** [review]
The code looks good to me well structured it is had some small queries do clarify them after merging

**Comment by dev-enforced** [comment]
image
View of The badges  look like these
image
The content of the button seems to be overflowing

File: products-page/products-page.css (hunk 1 above)
Comment (shuklahimanshu81) [reply products-page/products-page.css]: To overwrite styling of card used in CL.

File: products-page/products-page.css (hunk 2 above)
Comment (shuklahimanshu81) [reply products-page/products-page.css:62]: Thanks for pointing will change while refactoring.

**Comment by shuklahimanshu81** [comment]
> Added products to product-page.



> image View of The badges look like these image The content of the button seems to be overflowing

Fixed the issue.

**Review by nshanbhag20001412 — APPROVED** [review]
looks good to me

**Pull request merged**